### DISCUSSION TITLE
Icon Request: fa-doughnut-chart

### DISCUSSION THREAD

**Discussion by amayd** [FortAwesome/Font-Awesome#21160] [Icon Requests]
Hi Please provide with doughnut chart icon. 

![dougnut](https://cloud.githubusercontent.com/assets/2774503/10110989/98b10810-6385-11e5-9f48-d764160b0c31.png)

**Comment by tagliala** [comment]
@MKTGExperts thanks for your example

**Comment by davidwieler** [comment]
Second this. Would be an awesome addition

**Comment by todordev** [comment]
+1

**Comment by amayd** [comment]
any update for the same

**Comment by mdolberg** [comment]
sounds great if u could add it

**Comment by M3nace** [comment]
+1

**Comment by G-Driver** [comment]
+1

**Comment by ChriBack** [comment]
+1

**Comment by JMBratten** [comment]
+1

**Comment by leonardodelfino** [comment]
+1 for those icons

**Comment by acmeff** [comment]
+1

**Comment by dexterpeters** [comment]
+1

**Comment by alevato** [comment]
+1

**Comment by galvakojis** [comment]
+1

**Comment by rsiebeling** [comment]
+1

**Comment by harshalc** [comment]
+1

**Comment by laurelthorburn** [comment]
Does font awesome have a donut icon? I didn't see one in there when I checked this year

**Comment by pnajman-modoolar** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
